samples/speech/index.html

```diff
@@ -46,8 +46,11 @@
 
         <div id="BotChatGoesHere"></div>
 
-        <script src="../../botchat.js"></script>
-
+        <script src="../../BotChat.js"></script>
```

Comment:
ok. So as we can see, the library is BotChat while the file name is botchat. when webpack, i need the [name]...Let me see if there is a way to compromise that...